infrastructure/cdn-in-a-box/variables.env

```diff
@@ -122,3 +122,5 @@ AUTO_SNAPQUEUE_ENABLED=true
 AUTO_SNAPQUEUE_SERVERS=trafficops,trafficops-perl,trafficmonitor,trafficrouter,trafficvault,edge,mid-01,mid-02
 AUTO_SNAPQUEUE_POLL_INTERVAL=2
 AUTO_SNAPQUEUE_ACTION_WAIT=2
+LOAD_TRAFFIC_OPS_DATA=${LOAD_TRAFFIC_OPS_DATA:-true}
```

Comment:
These don't appear to be evaluating as if this were a bash script. This variable is getting set to literal `'${LOAD_TRAFFIC_OPS_DATA:-true}'`

Reply:
Where are you seeing `LOAD_TRAFFIC_OPS_DATA` being set to literal `${LOAD_TRAFFIC_OPS_DATA:-true}`? I tested on macOS with Docker Desktop and on Linux and was unable to reproduce. Testcase:

```shell
docker-compose run --rm edge sh -c 'echo $LOAD_TRAFFIC_OPS_DATA'
```
echos `true` for me.

Reply:
I added an echo statement to the trafficops-init.sh script: 
```
echo "LOAD_TRAFFIC_OPS_DATA = $LOAD_TRAFFIC_OPS_DATA. Skipping loading of traffic ops data."
```
output:
```
+ echo 'LOAD_TRAFFIC_OPS_DATA = ${LOAD_TRAFFIC_OPS_DATA:-true}. Skipping loading of traffic ops data.'
LOAD_TRAFFIC_OPS_DATA = ${LOAD_TRAFFIC_OPS_DATA:-true}. Skipping loading of traffic ops data.
```

When I run that command you pasted, I don't see any output. Maybe a difference in docker-compose versions? 
```
$ docker-compose version
docker-compose version 1.18.0, build 8dd22a9
docker-py version: 2.6.1
CPython version: 3.6.8
OpenSSL version: OpenSSL 1.0.2k-fips  26 Jan 2017

$ docker version
Client:
 Version:           18.06.3-ce
 API version:       1.38
 Go version:        go1.10.3
 Git commit:        d7080c1
 Built:             Wed Feb 20 02:26:51 2019
 OS/Arch:           linux/amd64
 Experimental:      false

Server:
 Engine:
  Version:          18.06.3-ce
  API version:      1.38 (minimum version 1.12)
  Go version:       go1.10.3
  Git commit:       d7080c1
  Built:            Wed Feb 20 02:28:17 2019
  OS/Arch:          linux/amd64
  Experimental:     false

$ cat /etc/centos-release
CentOS Linux release 7.4.1708 (Core)
```

Reply:
Looking at https://docs.docker.com/compose/env-file/ it doesn't seem to indicate that variable interpolation can be done in a .env file. It seems like it only accepts `VAR=VAL`. Maybe you have that variable set in your environment already?

Reply:
From my testing, parameter expansion only works in .env files in docker/compose@6d2658ea65 and up (stable docker-compose versions 1.26.0 and up), when docker-compose started using `python-dotenv` to process .env files.

Specifying those environment variables in `docker-compose.yml` in 4b7d9a0a4e.